tests/testthat/test-metadata.R

```diff
@@ -759,10 +759,12 @@ test_that("xportr_*: Domain is kept in between calls", {
 # end
 
 test_that("`xportr_metadata()` results match traditional results", {
+
+  if (requireNamespace("magrittr", quietly = TRUE)) {
```

Comment:
`{magrittr}` is on `Imports` in the `DESCRIPTION` file.

We can safely assume it is installed and remove all of these `requreNamespace("magrittr", ...)`